cmd/osctl/cmd/config.go

```diff
@@ -138,6 +140,20 @@ var configGenerateCmd = &cobra.Command{
 }
 
 func genV1Alpha1Config(args []string) {
+	// If output dir isn't specified, set to the current working dir
+	var err error
+	if outputDir == "" {
+		outputDir, err = os.Getwd()
+		if err != nil {
+			helpers.Fatalf("failed to get working dir: %w", err)
+		}
+	}
+
+	// Create dir path, ignoring "already exists" messages
+	if err = os.MkdirAll(outputDir, os.ModePerm); err != nil && !os.IsExist(err) {
```

Comment:
We may need to invert this. `!os.IsExist(err)` doesn't guarantee that it does indeed exist. This is one of the caveats with `os.IsExist`. There could be a valid error, maybe something like permission denied or something like that.

Reply:
Whoops. I messed up. I was turned upside down.